central/main.go

```diff
@@ -820,6 +821,23 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
 		},
 	)
 
+	repoMappingRoute := "/api/extensions/scanner-v4/repomappings"
```

Comment:
Nit: I prefer `repository-mappings` or just `mappings`.

Are you planning to have one endpoint for each bundle type? If so, can you explain why? My initial expectation is one endpoint where you can specify the files you're looking for (params, path, etc.) in the URL.